tests/cypress/e2e/for-teacher_page/customize_class_page/drag_and_drop.cy.js

```diff
@@ -0,0 +1,77 @@
+import {loginForTeacher} from '../../tools/login/login.js'
+import { createClass } from "../../tools/classes/class";
+
+describe('Back to class button', () => {
+  it('passes', () => {
+    loginForTeacher();
+    cy.wait(500);
```

Comment:
Do we needs this `wait` ? This always looks a bit scary to me, because if we're ever on a bit slower computer in which this takes 600ms, the test fails. Is it necessary?

Cypress best practices have something to say about this: https://docs.cypress.io/guides/references/best-practices#Unnecessary-Waiting

Reply:
You are right, I upload them as an error, because they make it easier see the test in action, but in these cases there's no reason to keep the wait.